hazelcast/src/main/java/com/hazelcast/sql/SqlColumnType.java

```diff
@@ -67,7 +67,12 @@
     TIMESTAMP_WITH_TIME_ZONE(OffsetDateTime.class),
 
     /** OBJECT type, could be represented by any Java class. */
-    OBJECT(Object.class);
+    OBJECT(Object.class),
+
+    /**
+     * The type of the generic SQL {@code NULL} literal.
```

Comment:
Can we add a note to JavaDoc that returned value is always `null`?

Reply:
fixed